src/migrations/20240111125100-automated-actions.js

```diff
@@ -0,0 +1,43 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS action_sets
+        (
+            id SERIAL PRIMARY KEY NOT NULL,
+            created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+            created_by_user_id INTEGER,
+            name varchar(255),
+            actor_id INTEGER,
+            source varchar(255),
+            source_id INTEGER,
+            payload JSONB DEFAULT '{}'::jsonb NOT NULL,
+            FOREIGN KEY (project) references projects(id) ON DELETE CASCADE,
```

Comment:
`project` needs to be added as a column in the table before defining it as a foreign key.

Reply:
Yes! :D